aws/resource_aws_efs_file_system_test.go

```diff
@@ -86,7 +86,7 @@ func TestAccAWSEFSFileSystem_basic(t *testing.T) {
 					resource.TestMatchResourceAttr(
 						"aws_efs_file_system.foo",
 						"dns_name",
-						regexp.MustCompile("^[^.]+.efs.us-west-2.amazonaws.com$"),
+						regexp.MustCompile("^[^.]+.efs.([0-9A-Za-z]+-[0-9A-Za-z]+-[0-9]).amazonaws.com$"),
```

Comment:
Maybe `([a-z]+-[a-z]+-[0-9])` is enough for the regex?
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

Reply:
Yes, good point. I have updated the pull request to use this `regexp.MustCompile("^[^.]+.efs.([a-z]+-[a-z]+-[0-9]).amazonaws.com$")`

```
TF_ACC=1 go test ./aws -v -run=TestAccAWSEFSFileSystem_basic -timeout 120m
go: finding github.com/terraform-providers/terraform-provider-tls v2.1.0+incompatible
go: finding github.com/terraform-providers/terraform-provider-tls v2.1.0+incompatible
=== RUN   TestAccAWSEFSFileSystem_basic
=== PAUSE TestAccAWSEFSFileSystem_basic
=== CONT  TestAccAWSEFSFileSystem_basic
--- PASS: TestAccAWSEFSFileSystem_basic (85.55s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       86.648s
```